docs/widgetbook-cloud/hosting.mdx

```diff
@@ -37,3 +37,111 @@ To create a Widgetbook Build, follow these steps inside your Widgetbook project:
    ```bash
    widgetbook cloud build push --api-key PROJECT_API_KEY
    ```
+
+<Info>
+The `cloud push` command uses the following directories:
+  1. `build/web/`: to create a `.zip` archive that will be uploaded to Widgetbook Cloud.
+  2. `.dart_tool/build/generated/[your_app_name]/`: to send metadata, _about the generated use-cases_, that will be used for [Widgetbook Cloud Review](/widgetbook-cloud/reviews). 
+</Info>
+
+## Pushing new builds with CI
+
+To better integrate Widgetbook Cloud with your development workflow, it is recommended to use CI/CD to push a new build **on every push** of a new commit.
+
+<Info>
+The following workflows assume you have Widgetbook structured as a sub-package
+
+```tree
+your_app/
+├── pubsepc.yaml
+├── lib/
+└── widgetbook/
+    ├── pubsepc.yaml
+    └── lib/
+```
+</Info>
+
+<Tabs
+  groupId="ci"
+  values={[
+    { label: "GitHub Actions", value: "github" },
+    { label: "Codemagic", value: "codemagic" },
+  ]}
+>
+  <TabItem value="github">
+    ```yaml
+    name: Widgetbook Cloud Hosting
+    on: push
+
+    jobs:
+      widgetbook-cloud-hosting:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout
+            uses: actions/checkout@v3
+
+          - name: Setup flutter
+            uses: subosito/flutter-action@v2
+            with:
+              channel: stable
+
+          - name: Bootstrap App
+            run: |
+              flutter pub get
+              # Add any other steps needed to make your
+              # app widgets available for Widgetbook
+
+          - name: Build Widgetbook
+            working-directory: widgetbook
+            run: |
+              flutter pub get
+              dart run build_runner build -d
+              flutter build web -t lib/main.dart
+
+          - name: Install Widgetbook CLI
+            run: dart pub global activate widgetbook_cli
+
+          - name: Push Widgetbook Build
+            working-directory: widgetbook
+            run: widgetbook cloud build push --api-key ${{ secrets.WIDGETBOOK_API_KEY }}
+    ```
+  </TabItem>
+
+  <TabItem value="codemagic">
+    ```yaml
+    workflows:
+      widgetbook-cloud-hosting:
+        name: Widgetbook Cloud Hosting
+
+        triggering:
+          events:
+            - push
+
+        environment:
+          flutter: stable
+          groups:
+            - widgetbook_credentials
```

Comment:
This part implicitly defines `WIDGETBOOK_API_KEY` used below. Can we expect readers to understand this?

Reply:
It's the same case with GitHub secrets,
Maybe we should either add comments to both of them (to be beginner friendly), or we leave both implicitly used.
What do you think?

Reply:
I've added some guides to how to do so on both GH Actions and Codemagic.